Upstream-Status: Pending
---
diff --git a/src/beam_search_scorer.cpp b/src/beam_search_scorer.cpp
index 2b846636..920b32ac 100644
--- a/src/beam_search_scorer.cpp
+++ b/src/beam_search_scorer.cpp
@@ -133,7 +133,7 @@ void BeamSearchScorer::Process(Sequences& sequences,
         auto clone = hypothesis_buffer_.Span().subspan(hypothesis_buffer_used_, src.size());
         hypothesis_buffer_used_ += clone.size();
 
-        copy(cpu_span{src}, cpu_span{clone});
+        copy(cpu_span<const int32_t>{src}, cpu_span<int32_t>{clone});
         beam_hyp.Add(clone, next_score);
       } else {
         // Add next predicted token since it is not eos_token.
@@ -189,7 +189,7 @@ void BeamSearchScorer::Finalize(Sequences& sequences,
       std::span<const int32_t> src = sequences.GetSequence(batch_beam_index).Span();
       auto clone = hypothesis_buffer_.Span().subspan(hypothesis_buffer_used_, src.size());
       hypothesis_buffer_used_ += clone.size();
-      copy(cpu_span{src}, cpu_span{clone});
+      copy(cpu_span<const int32_t>{src}, cpu_span<int32_t>{clone});
       beam_hyp.Add(clone, final_score);
     }
   }
