Upstream-Status: Pending
---
diff --git a/cmake/ortlib.cmake b/cmake/ortlib.cmake
index 328877b9..cbe30f7c 100644
--- a/cmake/ortlib.cmake
+++ b/cmake/ortlib.cmake
@@ -16,7 +16,7 @@ if(ORT_HOME)
     set(ORT_HEADER_DIR ${ORT_HOME}/include/onnxruntime)
     set(ORT_LIB_DIR ${ORT_HOME}/lib)
   else()
-    set(ORT_HEADER_DIR ${ORT_HOME}/include)
+    set(ORT_HEADER_DIR ${ORT_HOME}/include/onnxruntime)
     set(ORT_LIB_DIR ${ORT_HOME}/lib)
   endif()
 else()
